walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,55 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    ArrayList<Event> list = new ArrayList(events);
+    Collections.sort(list, (eventA, eventB) -> TimeRange.ORDER_BY_START.compare(eventA.getWhen(), eventB.getWhen()));
```

Comment:
can we use Arrays.sort(events.toArray(), ...) here? this way we dont need to have extra line to convert the list to events